src/mono/sample/mbr/browser/main.js

```diff
@@ -1,4 +1,4 @@
-import { dotnet } from './dotnet.js'
+import { dotnet } from './_framework/dotnet.js'
```

Comment:
So anyone consuming the runtime needs to hardcode _framework into their paths now, I guess this is an official part of the ABI?

Reply:
Yes, changing layout is breaking change for non-blazor templates.
I'm OK with it.
cc @radical @lewing

Reply:
I think we should be as a close to blazor as possible. Since non-blazor templates are "experimental", I wouldn't about it much, right?

Reply:
We should be able to get to blazor by setting some properties, but the default should not be blazor. We do have the browser/console templates.